### PR TITLE
[occm] update doc for some easy to miss parts

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:
update some doc for additional tips which is easy to make mistake

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/using-openstack-cloud-controller-manager.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The following guide has been tested to install Kubernetes v1.17 on Ubuntu 18.04.
 
 ### Steps
 
-- Create the kubeadm config file according to [`manifests/controller-manager/kubeadm.conf`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/kubeadm.conf)
+- Create the kubeadm config file according to [`manifests/controller-manager/kubeadm.conf`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/kubeadm.conf), you need modify the kubeadm version configuration according to your settings.
```
Comment (lingxiankong) [inline docs/using-openstack-cloud-controller-manager.md:36]: The content you added is still not very clear to readers, what's the motivation of the addition?

File: docs/using-openstack-cloud-controller-manager.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ The following guide has been tested to install Kubernetes v1.17 on Ubuntu 18.04.
     kubectl create secret -n kube-system generic cloud-config --from-file=cloud.conf
     ```
 
-- Create RBAC resources and openstack-cloud-controller-manager deamonset.
+- Create RBAC resources and openstack-cloud-controller-manager deamonset. Optionaly if you use certs then put the cert file into folder `/etc/ssl/certs/`, and update `ca-file` in configuration file `cloud.conf` above accordingly.
```
Comment (lingxiankong) [inline docs/using-openstack-cloud-controller-manager.md:52]: It's a little bit late to mention 'updating ca-file' here after the config secret already being created, we could reference https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md#global in step 4 and add more description for the option `ca-file` e.g. 

```
`ca-file` Optional. CA certificate bundle file for communication with Keystone service, this is required when using the https protocol in the Keystone service URL. openstack-cloud-controller-manager should have access to the file location.
```

File: docs/using-openstack-cloud-controller-manager.md (hunk 1 above)
Comment (jichenjc) [reply docs/using-openstack-cloud-controller-manager.md:36]: I actually copied the contents but 
the version in that file doesn't fit for my k8s version
just a reminder for someone to try this, they need change the version accordindly

File: docs/using-openstack-cloud-controller-manager.md (hunk 2 above)
Comment (jichenjc) [reply docs/using-openstack-cloud-controller-manager.md:52]: ok, I will see what I can do 
at least, my env is https so I think it's helpful to folks who use https env :)

**Comment by jichenjc** [comment]
@lingxiankong can you help to check updated one? Thanks

File: docs/using-openstack-cloud-controller-manager.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ The following guide has been tested to install Kubernetes v1.17 on Ubuntu 18.04.
 
 - Bootstrap worker nodes. You need to set `--cloud-provider=external` for kubelet service before running `kubeadm join`.
 
-- Create a secret containing the cloud configuration. You can find an example config file in [`manifests/controller-manager/cloud-config`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-config). Save the configuration to a file named *cloud.conf*, then:
+- Create a secret containing the cloud configuration. You can find an example config file in [`manifests/controller-manager/cloud-config`](https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-config). If you have certs you need put the cert file into folder `/etc/ssl/certs/` and update the update `ca-file` in the configuration file, refer to [ca-file section](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md#global) for furthe info. After that, Save the configuration to a file named *cloud.conf*, then:
```
Comment (lingxiankong) [inline docs/using-openstack-cloud-controller-manager.md:46]:
```
refer to `ca-file` option [here](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md#global) for further information. 
```

Comment (jichenjc) [reply docs/using-openstack-cloud-controller-manager.md]: ok, updated

**Pull request merged**